tool/dart2_test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+# Temporary workaround until Pub supports --preview-dart-2 flag
+set -e
+for filename in test/*_test.dart; do
+    dart --preview-dart-2 --enable_asserts "$filename"
```

Comment:
Let's `echo` the name of the file we're running. Alternatively, `set -x` should have roughly the same effect.